stdlib/rational.gr

```
@@ -23,13 +23,39 @@ from Numbers use {
 
 provide { fromNumber, toNumber, numerator, denominator }
 
+/**
+ * Gets the numerator and denominator of the rational as a tuple
+ *
+ * @param x: The rational to convert
+ * @returns The numerator and denominator of the rational
+ *
+ * @since v0.6.0
+ */
+@unsafe
+provide let toIntegerRatio = (x: Rational) => {
+  (numerator(x), denominator(x))
+}
+
+/**
+ * Makes a rational from the numerator and denominator
+ *
+ * @param numerator: The numerator
+ * @param denominator: The denominator
+ * @returns The built rational
+ *
+ * @since v0.6.0
+ */
+provide let fromIntegerRatio = (numerator: Number, denominator: Number) => {
+  fromNumber(numerator / denominator)
```

I think this should throw an exception if the inputs are not integers